packages/app/src/components-styled/table.ts

```diff
@@ -17,6 +17,7 @@ import {
   width,
   WidthProps,
 } from 'styled-system';
+import { StyledShouldForwardProp } from '~/utils/styledShouldForwardProp';
```

Comment:
We write all file names in kebab-case. Also it seems like you are exporting the function starting with a capital letter as if it was a type, but it is used as a value here. So I think this should be 

```suggestion
import { styledShouldForwardProp } from '~/utils/styled-should-forward-prop';
```

Reply:
right, this makes sense. I just glanced a the other util files so I missed that there was a mix of casings. The requested updates have been made 👍